### PR TITLE
Examples of design systems to the Parts Ex

### PR DESCRIPTION
**Pull request by varya**

### CODE REVIEW

File: slides/ds101-worskhop.mdx [hunk 1]
```diff
@@ -428,9 +428,19 @@ to find out the relevant parts of your design system.
 
 ---
 
-- [Carbon design system](https://www.carbondesignsystem.com)
-- [Polaris](https://polaris.shopify.com/)
-- [Zendesk Garden](https://garden.zendesk.com/)
+<Figure
+  src="/assets/parts-carbon.png"
```
Comment (okonet) [inline slides/ds101-worskhop.mdx:432]: `alt` is missing and it is required. Please add it.

File: slides/ds101-worskhop.mdx [hunk 2]
```diff
@@ -428,9 +428,19 @@ to find out the relevant parts of your design system.
...
+  caption={
+    <a href="https://www.carbondesignsystem.com/">Carbon Design System</a>
+  }
+/>
+
+---
+
+<Figure
```
Comment (okonet) [inline slides/ds101-worskhop.mdx:441]: Same `alt` is missing

**Review by okonet — CHANGES_REQUESTED** [review]

File: slides/ds101-worskhop.mdx (hunk 1 above)
Comment (varya) [reply slides/ds101-worskhop.mdx:432]: Added

File: slides/ds101-worskhop.mdx (hunk 2 above)
Comment (varya) [reply slides/ds101-worskhop.mdx:441]: Added

**Review by okonet — APPROVED** [review]

**Pull request merged**